client/app/account/user/security/2fa/user-security-2fa.routing.js

```diff
@@ -0,0 +1,12 @@
+angular
+  .module('UserAccount')
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider.state('app.account.user.security.mfa', {
+      url: '/mfa',
+      templateUrl: 'account/user/security/2fa/user-security-2fa.html',
+      controller: 'UserAccount.controllers.doubleAuth.2fa.enable',
```

Comment:
A component would've been nice

Reply:
It would be. It's not the main goal of this feature. Should be handle by a global refactor.